webapp/src/js/enketo/widgets/notewidget.js

```diff
@@ -36,9 +36,9 @@ define( function( require, exports, module ) {
      * @param {*=} e     event
      */
 
-    function Notewidget( element, options ) {
+    function Notewidget( element ) {
         this.namespace = pluginName;
-        Widget.call( this, element, options );
+        this.element = element;
```

Comment:
All the other widgets call `Object.assign( this, new Widget( element, options ) );` - why not here?

Reply:
The notewidget has been deleted - now we do markdown in API.